Applications/Games/Subnautica/resources/fix.reg

```diff
@@ -3,8 +3,8 @@ REGEDIT4
 [HKEY_CURRENT_USER\Software\Wine\X11 Driver]
 "UseTakeFocus"="N"
 
-[HKEY_CURRENT_USER\Software\Wine\AppDefaults\Steam.exe]
-"Version"="winxp64"
- 
-[HKEY_CURRENT_USER\Software\Wine\AppDefaults\steamwebhelper.exe]
-"Version"="winxp64"
+[HKEY_CURRENT_USER\Software\Wine]
+"HideWineExports"="Y"
+
+[HKEY_CURRENT_USER\Software\Wine\Direct3D]
+"csmt"="0x00000000"
```

Comment:
Use the csmt plugin instead.

Reply:
CSMT was used to increase performance in some applications. Its now depreciated.